mnelab/model.py

```diff
@@ -598,11 +601,20 @@ def get_compatibles(self):
     @data_changed
     def append_data(self, names):
         """Append the given raw data sets."""
+        files = [self.current["data"]]
         for d in self.data:
             if d["name"] in names and d["data"] is not None:
```

Comment:
I was wondering when `d["data"]` can be `None`...

Reply:
Oops, this comes from an older version.